manifests/charts/istio-operator/templates/deployment.yaml

```diff
@@ -37,7 +37,7 @@ spec:
             runAsGroup: 1337
             runAsUser: 1337
             runAsNonRoot: true
-          imagePullPolicy: IfNotPresent
+          imagePullPolicy: Always
```

Comment:
This actually impacts our CI system, as integ tests may not pull the latest image when executing.  Ideally, I'd link it to the same variable that pilot uses, but the manifests system didn't like that, for reasons I don't understand.

Reply:
In CI its pulling some local image like `istio-testing/manifest:istio-testing` I think so it probably doesn't impact in prow? Although certainly for local runs it would. I am a bit concerned about changing the main deployment to a not optimal default to help with local debugging. I agree it should use the test framworks --imagePullPolicy option, but we shouldn't change default behavior for operator

Reply:
I agree that Always always is not the right solution.  I'll open a tracking bug so we can hook this into the helm value for imagePullPolicy.

Reply:
Follw-up issue created: https://github.com/istio/istio/issues/36054